source/database-connections/save-relational-connection.txt

```diff
@@ -0,0 +1,85 @@
+
+.. _rm-save-relational-connection:
+
+============================================
+Save a Relational Database Connection
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can save a new relational database connection from the Relational Migrator
+:guilabel:`Connections` page, or when creating or modifying a data migration job.
+
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Add a connection
+      
+      In the list of :guilabel:`Relational` connections, click :guilabel:`+ Add connection`.
```

Comment:
As with the MonogDB topic, the procedure should be valid regardless of which of the 4 entry points was chosen.